lib/camunda-cloud/Modeler.js

```diff
@@ -60,7 +62,8 @@ Modeler.prototype._camundaCloudModules = [
   rulesModule,
   zeebePropertiesProviderModule,
   cloudElementTemplatesPropertiesProvider,
-  zeebeModdleExtension
+  zeebeModdleExtension,
+  iconRendererModule
```

Comment:
To consider: Also add this to _Cloud Viewer_, _Cloud Navigated Viewer_, if we provide such things.

We want icons to be visible, end-to-end (cf. https://github.com/bpmn-io/internal-docs/issues/445).

Reply:
Generally yes, but afaik camunda-bpmn-js does not provide Viewers. Viewer consumers so far just use bpmn-js vanilla.

Considering how we bundle the custom renderer, we might want to change this, and offer a camunda-bpmn-js viewer as well.

Reply:
Let us leave it as is and create a follow up for this FR.

Reply:
https://github.com/camunda/camunda-bpmn-js/issues/110